Header header
string group_name
geometry_msgs/Pose pose
float32[3] pointing_vector
float32[3] facing_vector
float32 radius
int64 capable_candidates

